### PR TITLE
move docs to website

### PR DESCRIPTION
**Pull request by doodyparizada**
Start writing docs here: https://github.com/kinecosystem/partners-docs/

### CODE REVIEW

**Comment by OrenZak** [comment]
@doodyparizada sync with dev

**Review by OrenZak — APPROVED** [review]
We should keep the License here also, other than that 👍

File: README.md [hunk 1]
```diff
@@ -4,168 +4,4 @@
...
-
-* [Misc](docs/MISC.md)
-
-
-## Common Errors ##
-The Ecosystem APIs can response with few types of error, [learn more here](docs/COMMON_ERRORS.md)
-
-## License ##
-
-The ```kin-ecosystem-android-sdk``` library is licensed under the MIT license.
```
Comment (OrenZak) [inline README.md:170]: we should keep that.

**Comment by doodyparizada** [comment]
@OrenZak Added license

**Comment by doodyparizada** [comment]
@OrenZak Image has been replaced

**Pull request merged**